### PR TITLE
Keeper > RegistrySynchronizer: Make SyncUpkeepQueue size configurable

### PR DESCRIPTION
**Pull request by b-s-d**
The registry synchronizer currently caps parallel upkeep sync routines to 10. This value should be configurable for node operators. Create an env var (default 10) and use this instead of the fixed constant.

See story for more details: https://app.shortcut.com/chainlinklabs/story/9028/make-synchupkeepqueue-size-configurable

### CODE REVIEW

**Comment by b-s-d** [comment]
@RyanRHall : Please let me know if it makes sense to reflect the config changes in the [Wiki](https://github.com/smartcontractkit/chainlink/wiki/How-to-Run-Keeper-Jobs) and  [Node Operators documentation](https://github.com/smartcontractkit/documentation/blob/f17c384788f0c01dc53da76f64ba43a3b577cb3f/docs/Node%20Operators/configuration-variables.md) once we have this PR merged, thanks.

File: core/services/keeper/registry_synchronizer_sync.go [hunk 1]
```diff
@@ -12,9 +12,6 @@ import (
 	"github.com/smartcontractkit/chainlink/core/utils"
 )
 
-// syncUpkeepQueueSize represents the max number of upkeeps that can be synced in parallel
-const syncUpkeepQueueSize = 10
```
Comment (RyanRHall) [inline core/services/keeper/registry_synchronizer_sync.go:16]: :clap:

**Review by RyanRHall — DISMISSED** [review]

**Comment by RyanRHall** [comment]
Looks great @b-s-d.

File: core/store/config/schema.go [hunk 2]
```diff
@@ -108,6 +108,8 @@ type ConfigSchema struct {
 	KeeperRegistryCheckGasOverhead             uint64                        `env:"KEEPER_REGISTRY_CHECK_GAS_OVERHEAD" default:"200000"`
 	KeeperRegistryPerformGasOverhead           uint64                        `env:"KEEPER_REGISTRY_PERFORM_GAS_OVERHEAD" default:"150000"`
 	KeeperRegistrySyncInterval                 time.Duration                 `env:"KEEPER_REGISTRY_SYNC_INTERVAL" default:"30m"`
+	KeeperRegistrySyncUpkeepQueueSize          uint32                        `env:"KEEPER_REGISTRY_SYNC_UPKEEP_QUEUE_SIZE" default:"10"`
+	Layer2Type                                 string                        `env:"LAYER_2_TYPE"`
```
Comment (RyanRHall) [inline core/store/config/schema.go:112]: is this supposed to be here?

**Review by RyanRHall — APPROVED** [review]

**Pull request merged**